src/modules/app.js

```diff
@@ -0,0 +1,180 @@
+import '../index.css';
+import logo from '../assets/burger-logo.png';
+import UserInterface from './userInterface.js';
+import MealAPI from './mealAPI.js';
+import CurrentCategory from './currentCategory.js';
+import CurrentMeal from './currentMeal.js';
+import InvolvementAPI from './involvementAPI.js';
+import PopUp from './popUp.js';
+import Menu from './menu.js';
+
+class App {
+  start() {
+    this.imageContainer = document.querySelectorAll('.image');
+    this.asyncEnv();
+  }
+
+  setLogo() {
+    Array.from(this.imageContainer).forEach((container) => {
+      const logoImg = new Image();
+      logoImg.src = logo;
+      logoImg.classList.add('hero__logo');
+      logoImg.alt = 'Yummy Recipes Logo';
+      container.appendChild(logoImg);
+    });
+  }
+
+  async asyncEnv() {
+    this.setLogo();
+    const categories = await MealAPI.getCategories();
+    Menu.displayCategories(categories);
+    UserInterface.displayCategories(categories);
+    UserInterface.counterText('Categories');
+    this.selectTags();
+    this.setListeners();
+  }
+
+  goBackToCommonParent = (node, a) => {
+    if (
+      node.getAttribute(a.type).includes(a.item)
+      !== node.parentNode.getAttribute(a.type).includes(a.item)
+    ) return node;
+    return this.goBackToCommonParent(node.parentNode, a);
+  };
```

Comment:
[OPTIONAL] Great piece of code! I would recommend using a meaningful name for `a` this way it would be easier to understand by other developers.